src/test/java/uk/gov/justice/hmpps/prison/service/AdjudicationsServiceTest.java

```diff
@@ -2085,7 +2085,7 @@ public void updateSanctions() {
             assertThat(saveCapture.getValue().getOffenderBookId()).isEqualTo(200L);
             assertThat(saveCapture.getValue().getSanctionSeq()).isEqualTo(7L);
             assertThat(saveCapture.getValue().getOicSanctionCode()).isEqualTo(OicSanctionCode.ADA);
-            assertThat(saveCapture.getValue().getCompensationAmount()).isEqualTo(new BigDecimal("1000.0"));
+            assertThat(saveCapture.getValue().getCompensationAmount()).isEqualByComparingTo("1000.0");
```

Comment:
Would it be easier to understand if we then used pounds/pence - so compared to 1000.00